.gitignore

```diff
@@ -8,3 +8,4 @@ yarn-debug.log*
 yarn-error.log*
 test/unit/coverage
 packages/*/dist
+.idea
```

Comment:
I recommend doing this in your local `.gitignore` file and not including it in projects.

```bash
echo ".idea" >> ~/.gitignore
git config --global core.excludesfile ~/.gitignore
```

Reply:
Good one, reverted this change!